projects/core/components/theme-night/theme-night.style.less

```diff
@@ -31,6 +31,7 @@
     --tui-neutral-fill: rgb(149, 155, 164);
     --tui-neutral-bg: rgba(149, 155, 164, 0.32);
     --tui-neutral-bg-hover: rgba(149, 155, 164, 0.48);
+    --tui-neutral-bg-active: rgba(149, 155, 164, 0.52);
```

Comment:
Such a variable is available in proprietary tokens. I think it's legit to add it here

Reply:
Should we add other status backgrounds for active states? Let's update status appearance too.

Reply:
"close" appearance removed, I'l better update css variables in a separate PR